exampleCourse/questions/element/checkbox/question.html

```diff
@@ -177,3 +177,29 @@
 
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-header">
+        Part 7
+    </div>
+
+    <div class="card-body">
+        <pl-question-panel>
+            <p>Here, letter keys are hidden with <code>hide-letter-keys="true"</code>. This option is especially useful when the answer options themselves are letters.</p>
+
+            <p>Group work provides which of the following benefits:</p>
+        </pl-question-panel>
+
+        <pl-checkbox answers-name="group_ans_hide_keys" partial-credit="true" partial-credit-method="EDC">
```

Comment:
This example doesn't seem to illustrate the new feature? Was this the wrong example? @tdy

Reply:
Oh weird, not sure what happened there. Fixed it now.